clustered/server/src/main/java/org/ehcache/clustered/server/EhcacheActiveEntity.java

```diff
@@ -742,6 +744,14 @@ private void destroyServerStore(ClientDescriptor clientDescriptor, DestroyServer
     boolean isDuplicate = isLifeCycleMessageDuplicate(destroyServerStore);
 
     if (!isDuplicate) {
+      ServerStoreImpl store = ehcacheStateService.getStore(name);
```

Comment:
I think this code should be moved into `ehcacheStateService.destroyServerStore(name);` 

It would be better to have all the code within the same class and classloader